### PR TITLE
chore: segregate renderer type for metrics

### PR DESCRIPTION
**Pull request by menduz**

### CODE REVIEW

File: src/kernel-loader/index.ts [hunk 1]
```diff
@@ -256,10 +256,8 @@ async function initKernel() {
     store.dispatch(setKernelError(error))
 
     // TODO: move this into a saga for setKernelError
-    trackCriticalError(error.error, error.extra)
-    if (error.level === 'fatal') {
-      disableAnalytics()
-    }
+    trackError(error.error, { context: 'kernel', ...(error.extra || {}) })
+    disableAnalytics()
```
Comment (nachomazzara) [inline src/kernel-loader/index.ts:260]: Should we need to disable analytics at any error level?

Comment (menduz) [reply src/kernel-loader/index.ts:267]: Yep, since the error produces a black screen of death. Will add a comment

**Review by nachomazzara — APPROVED** [review]
lgtm

File: src/integration/queryParamsConfig.ts [hunk 2]
```diff
@@ -2,4 +2,5 @@ const qs = new URLSearchParams(document.location.search || '')
 
 export const DEBUG_ANALYTICS = qs.has('DEBUG_ANALYTICS')
 export const ENV = qs.get('ENV')
-export const NETWORK = qs.get('NETWORK')
\ No newline at end of file
+export const NETWORK = qs.get('NETWORK')
+export const RENDERER_TYPE = qs.get('ws') ? 'native' : 'web'
```
Comment (cazala) [inline src/integration/queryParamsConfig.ts:6]: saaaaran 👀

**Pull request merged**